public/data/KendallDoesCoding.json

```diff
@@ -2,11 +2,22 @@
   "name": "Kendall Pinto",
   "type": "personal",
   "bio": "Hello, My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, JavaScript, and the one I'm currently working on projects with, HTML & CSS. I'm currently working on the project, A Very Mogul Christmas",
-  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "avatar": "https://github.com/KendallDoesCoding.png",
```

Comment:
@EddieHubCommunity/linkfree Please let me know which avatar looks better.

https://i.ibb.co/5LsjwVB/Avatar-Maker.png
https://github.com/KendallDoesCoding.png

The first one is more of a human kinda character, than the second. The second one just has my coding name written on it. I don't mind both, but wanna know your opinion on it.

Reply:
GitHub profile is preferable but it’s your choice.